src/Chirp.Web/Pages/Shared/_CheepList.cshtml

```diff
@@ -11,12 +10,13 @@
 <ul id="messagelist" class="cheeps">
     @foreach (CheepViewModel cheep in Model.Cheeps)
     {
+        Author cheepAuthor = AuthorRepository.GetAuthorByIdAsync(cheep.AuthorId).Result;
         <li>
             <div class="cheep-view">
                 <div class="cheep-info">
                     <strong>
                         <a href="/@cheep.Author">@cheep.Author</a>
-                        <p>Followers: @AuthorRepository.GetAuthorByName(cheep.Author).Followers.Count</p>
+                        <p>Followers: @cheepAuthor.Followers.Count</p>
```

Comment:
This doesn't reduce the amount of repository calls. You are just assigning a variable and using that instead of using the repository call return value directly

Reply:
It aggregates 2 different await AuthorRepository.GetAuthorByName calls into 1. Both for the Followers: @Authorrepository on line 47, and !Model.user.Following.Contains(await AuthorRepository.GetAuthorByIdAsync(cheep.AuthorId)) on line 34.